test/citestutils.js

```diff
@@ -199,6 +227,19 @@ function testRestore(params, inputStream, databaseName, callback) {
     }
   }
 
+  // Pipe via decryption if requested
+  if (params.encryption) {
+    if (params.useApi) {
+      callback(new Error('UNSUPPORTED: cannot abort API backups at this time.'));
```

Comment:
Wrong err msg here? `UNSUPPORTED: cannot perform encrypted API backups at this time.`